examples/SDM_example.py

```diff
@@ -0,0 +1,204 @@
+Device_Type = 124  # 124 = Stride & Distance Sensor
+Device_Number = 12345  # Change if you need.	
+Channel_Period = 8134
+Channel_Frequency = 57	
+
+# Fictive Config of Treammill 
+TreadmillSpeed = 2.777  # m/s => 10km/h
+TreadmillCadence = 160
+		
+##########################################################################
+
+class AntSendDemo:
+
+    def __init__(self):
+
+        self.ANTMessageCount = 0
+        self.ANTMessagePayload = array.array('B', [0, 0, 0, 0, 0, 0, 0, 0])
+
+		# Init Variables, needed
+        self.LastStrideTime = 0
+        self.StridesDone = 0
+        self.DistanceAccu = 0
+        self.Speed_Last = 0
+        self.TimeRollover = 0
+		
+        self.TimeProgramStart = time.time()
+        self.LastTimeEvent = time.time()
+
+	# Building up the Datapages
+		# This is just for demo purpose and can/will look diverent for every implementation
+    def Create_Next_DataPage(self):
+        # Define Variables
+        UpdateLatency_7 = 0
+		
+        self.ANTMessageCount += 1
+		
+        # Time Calculations
+        self.ElapsedSseconds = time.time() - self.LastTimeEvent
+        self.LastTimeEvent = time.time()	
+        UpdateLatency_7 += self.ElapsedSseconds # 1Second / 32 = 0,03125
+        UL_7 = int(UpdateLatency_7 / 0.03125)
+
+        # Stride Count, Accumulated strides.
+        # This value is incremented once for every two footfalls. 
+        StrideCountUpValue = 60.0 / (TreadmillCadence / 2.0) # In our Example 0,75
+		#print("LastStrideTime: ", self.LastStrideTime)	# jst for DEBUGGING
+        while (self.LastStrideTime > StrideCountUpValue):
+            self.StridesDone += 1
+            self.LastStrideTime-=StrideCountUpValue
+        self.LastStrideTime += self.ElapsedSseconds
+        if (self.StridesDone > 255):
+            self.StridesDone -= 255
+
+        # DISTANCE
+		# Accumulated distance, in m-Meters, Rollover = 256
+        self.DistanceBetween =  self.ElapsedSseconds * TreadmillSpeed 
+        self.DistanceAccu+=self.DistanceBetween # Add Distance beetween 2 ANT+ Ticks to Accumulated Distance
+        if (self.DistanceAccu > 255):
+            self.DistanceAccu -= 255
+        
+        self.distance_H = int(self.DistanceAccu) # just round it to INT
+        self.DistanceLow_HEX = int((self.DistanceAccu-self.distance_H)*16)
+        # erstmal zum Checken
+		#if(DistanceLow_HEX > 15) { printf("ERROR: Distance LOW HEX ueber 15: %d \n", DistanceLow_HEX); }
+        #printf("=> Distanz m: %d,%d \n", distance_H, DistanceLow_HEX)
+
+        # SPEED - Berechnung
+        self.var_speed_ms_H = int(TreadmillSpeed) # INT-Value
+        self.var_speed_ms_L = int(TreadmillSpeed*1000) - (self.var_speed_ms_H*1000) # Aus der Uebergabe km/h mit Kommawert machen => 20 = 2,0km/h
+        self.var_speed_ms_L_HEX = int((TreadmillSpeed-self.var_speed_ms_H)*256)
+
+        #TIME (erst am Ende da Veraenderung bei Distanz oder Zeit nur genommen werden <= laut Spezifikation)

```

Comment:
Try to keep comments in English :smiley:. Multiple places.

Reply:
yes, my fault.

done